HowToRPC.md

```diff
@@ -0,0 +1,166 @@
+directly, often using standard library functions.
+
+This protocol uses fixed URLs to specify the RPC endpoints, and passes
+request/response messages as HTTP request/response body using HTTP POST. It uses
+normal HTTP headers to pass the RPC metadata, such as System Parameters.
+
+### URL
+
+RPC URLs have the following format:
+
+```
+URL ::= BaseUrl "/" Service "/" Method
+```
+
+- **BaseUrl.** This is the base URL published by service owners, either via
+  documentation or service discovery. For most Google APIs, the BaseUrl looks
+  like `https://language.googleapis.com/$rpc` (experimental: this format may
+  change in the future). The base address can be found in the API reference
+  documentation where it is identified as the `Service name`. For this example,
+  `language.googleapis.com` is found in the Google Cloud Natural Language API
+  Reference.
+
+- **Service.** This is the fully qualified protobuf `service` name, such as
+  `google.cloud.language.v1.LanguageService`. In this case,
+  `google.cloud.language.v1` is the package name in
+  `google/cloud/language/v1/language_service.proto` and `LanguageService` is the
+  name of the service section found in this file.
+
+- **Method.** This is the protobuf `rpc` name, such as `AnnotateText`.
+
+### Requests
+
+RPC request messages are serialized and sent as the HTTP request body. For the
+server to handle the request properly, the client must set several HTTP request
+headers:
+
+- **Content-Type.** The request message format is specified by the Content-Type
+  header and must be `application/x-protobuf`.
+
+- **X-Goog-Api-Key.** This specifies a valid Google API key. It is optional if
+  the Authorization header is used.
+
+- **Authorization.** This specifies a valid Google OAuth access token in the
+  format of `Bearer {token}`. It is optional if an API key is provided.
+
+### Responses
+
+For successful requests, the HTTP status code is `200` and the HTTP response
+body contains the serialized RPC response message. For unsuccessful requests,
+the HTTP status code is the HTTP mapping for
+[`google.rpc.Code`](https://github.com/googleapis/googleapis/blob/master/google/rpc/code.proto)
+and the HTTP response body contains a serialized
+[`google.rpc.Status`](https://github.com/googleapis/googleapis/blob/master/google/rpc/status.proto)
+message. For details, see [Errors](https://cloud.google.com/apis/design/errors)
+in the [API Design Guide](https://cloud.google.com/apis/design).
+
+The HTTP response contains at least the following headers:
+
+- **Content-Type.** This specifies the response serialization format. For normal
+  responses and server errors, this will be `application/x-protobuf`. Different
+  values can be returned for network errors, such as when a message is rejected
+  by a network proxy. All such errors will be accompanied by appropriate HTTP
+  status codes.
+
+## [Examples](https://github.com/googleapis/googleapis.github.io/tree/master/examples/rpc/)
```

Comment:
Google APIs don't use plural words for directory name. I prefer we follow the tradition here.

Reply:
Is there a style guide or example of that? When I look at the [API Style Guide Naming Conventions](https://cloud.google.com/apis/design/naming_convention), I find

> Collection IDs should use plural form and lowerCamelCase, and American English spelling and semantics. For example: events, children, or deletedEvents.

Isn't this directory a collection of examples?